Makefile

```diff
@@ -357,6 +359,9 @@ dist_export:
 	@mkdir -p $(DIST_EXPORT_DIR)/test/functional
 	$(RSYNC) test/functional/* $(DIST_EXPORT_DIR)/test/functional
 
+.PHONY: locale_update
+locale_update: $(VENV)
+	$(VENV)/bin/python bin/locale_update.py
```

Comment:
Not sure if there's any particular history for the name locale_update, but the other `Makefile` targets seem to be actions, e.g., generate_changelog, update_version, upload_xpi. So perhaps this should be `update_locale`?